ground-ingest/src/main/java/edu/berkeley/ground/ingest/WikiConverter.java

```diff
@@ -0,0 +1,52 @@
+package edu.berkeley.ground.ingest;
+
+import java.lang.reflect.Type;
+import java.util.Map;
+
+import org.apache.avro.Schema;
+import org.apache.avro.generic.GenericData;
+import org.apache.avro.generic.GenericRecord;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonElement;
+import com.google.gson.reflect.TypeToken;
+
+import gobblin.configuration.WorkUnitState;
+import gobblin.converter.SingleRecordIterable;
+import gobblin.converter.ToAvroConverterBase;
+
+
+public class WikiConverter extends ToAvroConverterBase<String, JsonElement>{
```

Comment:
Is this a converter specifically for the Wikiepdia example? Why do we need specific converters?